cmd/ocsp-updater/main.go

```diff
@@ -398,20 +394,31 @@ func (updater *OCSPUpdater) revokedCertificatesTick(ctx context.Context, batchSi
 		return err
 	}
 
+	var purgeURLs []string
 	for _, status := range statuses {
-		meta, err := updater.generateRevokedResponse(ctx, status)
+		meta, respURLs, err := updater.generateRevokedResponse(ctx, status)
```

Comment:
I don't think `respURLs` makes sense as a variable name here. How about `allPurgeURLs` as the top-level variable, and `purgeURLs` as the inner variable?